<?xml version="1.0"?>

<launch>
  	<arg name="start_manager" default="true"/>
  	<arg name="nodelet_manager" default="image_flip_nodelet_manager"/>
	<arg name="robot" default="$(env ROBOT)"/>

	<node if="$(arg start_manager)" pkg="nodelet" type="nodelet" name="$(arg nodelet_manager)"  args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="CobKinectImageFlipNodelet" ns="/cob_image_flip" args="load cob_image_flip/CobKinectImageFlipNodelet /$(arg nodelet_manager)" output="screen">
	<rosparam command="load" file="$(find cob_image_flip)/ros/launch/image_flip_params.yaml"/>
	<param name="robot" type="string" value="$(arg robot)"/>
    <remap from="~pointcloud_in" to="/cam3d/rgb/points"/>
    <remap from="~pointcloud_out" to="/cam3d/upright/rgb/points"/>
    <remap from="~colorimage_in" to="/cam3d/rgb/image_color"/>
    <remap from="~colorimage_out" to="/cam3d/upright/rgb/image_color"/>
  </node>                 

</launch>
